modules/jpa-basic/src/main/java/com/devonfw/module/jpa/dataaccess/api/JpaHelper.java

```diff
@@ -74,4 +75,20 @@
     }
   }
 
+  /**
+   * This is the newer implementation of RevisionMetadataType should be used with
+   * {@link com.devonfw.module.basic.common.api.RevisionMetadata}
+   *
+   * @param revEntity die {@link AdvancedRevisionEntity}.
+   * @return die Instanz von {@link RevisionMetadataType} bzw. {@code null} falls {@code revision} den Wert {@code null}
+   *         hat.
+   */
+  public static RevisionMetadataType of(AdvancedRevisionEntity revEntity) {
```

Comment:
The static `of` method is typically used as a factory method of the same class defining the static method (as a short form for `valueOf` what is used inside the JDK as convention/pattern).
In this case `JpaHelper.of(entity)` the API is IMHO very confusing. You need to move the method to `RevisionMetadataType`. In case it should remain in `JpaHelper` you should rename this `of` method to `asRevisionMetadata` (like e.g. `asEntity`).